yaml-templates/generic-analysis-sonarcloud.yml

```diff
@@ -0,0 +1,42 @@
+# Generic Analysis with SonarCloud
+#
+# * If necessary, create an endpoint to connect to SonarCloud
+#
+# * Set the sonarCloudXXX variables below
+# 
+# * Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
```

Comment:
This is a template for generic analysis so it's confusing that the link is to ".../windows/dot-net". It might be better to reference a more generic page e.g. https://docs.microsoft.com/en-gb/azure/devops/pipelines/create-first-pipeline

Reply:
Done